### PR TITLE
[5.3] Code cleanup

### PR DESCRIPTION
**Pull request by danon**
It's not much, but I just want to see if people here are actually merging pull requests :)

### CODE REVIEW

**Review by browner12 — CHANGES_REQUESTED** [review]
imports should be done shortest to longest, so your change in `HandleQueuedMessage.php` will not be approved.

File: src/Illuminate/Queue/SyncQueue.php [hunk 1]
```diff
@@ -54,9 +54,9 @@ public function push($job, $data = '', $queue = null)
     /**
      * Handle an exception that occurred while processing a job.
      *
-     * @param  \Illuminate\Queue\Jobs\Job  $queueJob
-     * @param  \Exception  $e
-     * @return void
```
Comment (GrahamCampbell) [inline src/Illuminate/Queue/SyncQueue.php]: Please do not delete this.

File: src/Illuminate/Mail/Jobs/HandleQueuedMessage.php [hunk 2]
```diff
@@ -3,11 +3,10 @@
 namespace Illuminate\Mail\Jobs;
 
 use Closure;
-use Illuminate\Support\Str;
-use SuperClosure\Serializer;
-use Illuminate\Contracts\Queue\Job;
 use Illuminate\Contracts\Mail\Mailer;
 use Illuminate\Queue\SerializesAndRestoresModelIdentifiers;
+use Illuminate\Support\Str;
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Jobs/HandleQueuedMessage.php]: Please don't move these.

File: src/Illuminate/Database/Connectors/ConnectionFactory.php [hunk 3]
```diff
@@ -2,7 +2,6 @@
 
 namespace Illuminate\Database\Connectors;
 
-use PDO;
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Connectors/ConnectionFactory.php]: I think this was already fixed? Please rebase. :)

**Comment by GrahamCampbell** [comment]
Thanks. Left some comments for you.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by taylorotwell** [comment]
Some of these changes are invalid. Resubmit with corrections if you would like.

**Comment by PleaseDontKillMe** [comment]
I believe autoformat from PhpStorm moved these imports for me. I always used imports in alphabetical order (why shortest to longest?)

**Comment by browner12** [comment]
that's just the convention Laravel has taken.

**Comment by PleaseDontKillMe** [comment]
Does it matter, after all? Nothing bad happens if they're not in that "order" right? So why do you guys follow it?

**Comment by browner12** [comment]
no, it doesn't matter from a code execution standpoint.

why should we put them in alphabetical order? should we use spaces or tabs? should curly braces be on the same line or next?   none of these really matter, just a convention Laravel has picked to follow to make their code consistent.

**Comment by PleaseDontKillMe** [comment]
imports have nothing to do with code consistency. I use what PhpStorm autoformat enforces, you just click Ctrl+Alt+L and there you go. No more consistency problems.

**Comment by browner12** [comment]
and someone else's editor autoformats it a different way.

enough bikeshedding, focus on the important stuff.

**Comment by PleaseDontKillMe** [comment]
So why bother with it?

**Pull request closed without merging**